any/barcode-gas/barcode-gas-componenttype.md

```diff
@@ -0,0 +1,236 @@
+# Return component type from barcode
+
+This example shows how to extract the component material for a given gas distribution components encoded using the [ASTM F2897 standard](https://www.astm.org/Standards/F2897.htm). This specification defines requirements for the data used in the tracking and traceability base-62 encoding system and the format of the resultant code to characterize various components used in fuel gas piping systems.
+
+## Use cases
+
+In this example we are highlighting how to efficiently display the material for a natural gas component from a barcode captured in the field.  In the example below we are extracting the component material from the barcode string.  
+
+## Workflow
+
+Copy and paste the expression found in the expression template below to the Arcade editor in ArcGIS Online, the relevant location in ArcGIS Pro, or the relevant location in a custom app.
+
+To configure the script to your layer, you must replace the BARCODE variables with the field containing the barcode string from the natural gas component.  
+
+## Expression Template
+
+```js
+var BARCODE = $feature.BARCODE
```

Comment:
This is really picky, but none of the lines in this expression end in a semicolon. It's obviously not necessary, but we generally follow that rule in multi-line expressions to keep things clean and consistent.

Reply:
Added semicolons for consistency.